### PR TITLE
added files for school-type-eng beta, 

### PR DESCRIPTION
**Pull request by Jamie-Atkinson**
it has not been removed from alpha as still needed for school-eng register

should not be merged until register is ready to go live

### CODE REVIEW

File: data/beta/register/school-type-eng.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+fields:
+- school-type-eng
+- name
+- start-date
+- end-date
+phase: beta
+register: school-type-eng
```
Comment (gidsg) [inline data/beta/register/school-type-eng.yaml]: Does this have a `register-name`?

File: data/beta/register/school-type-eng.yaml [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+registry: department-for-education
+text: Types of schools in England
```
Comment (gidsg) [inline data/beta/register/school-type-eng.yaml]: Are you sure this is not `Types of school in England`?

File: data/beta/register/school-type-eng.yaml (hunk 1 above)
Comment (Jamie-Atkinson) [reply data/beta/register/school-type-eng.yaml]: School types in England

File: data/beta/school-type-eng/meta.yaml [hunk 3]
```diff
@@ -0,0 +1 @@
+register-name: School types in England
```
Comment (gidsg) [inline data/beta/school-type-eng/meta.yaml]: we should add a `custodian` here.

**Pull request merged**